script/lib/run-apm-install.js

```diff
@@ -4,7 +4,7 @@ const childProcess = require('child_process')
 
 const CONFIG = require('../config')
 
-module.exports = function (packagePath, ci) {
+module.exports = function (packagePath, ci, stderrOnly) {
```

Comment:
Food for thought: Without additional context, the name `stderrOnly` doesn't feel very expressive to me. How would you feel about changing it to something like `stdioOptions` with a default value of `'inherit'`?  And in the places that want to deviate from the default (e.g., the new code in `script/dev`), we can pass a custom value (e.g., `['inherit', 'pipe', 'inherit']`.

What do you think?

Reply:
Definitely better, I'll go with that!